privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -24,10 +25,28 @@ def __init__(
         :param dataset: dataset for training of shadow classifiers, test_data from dataset
         with concatenation [test_features, test_labels]
         """
+
+        if not (isinstance(dataset, tuple) and list(map(type, dataset)) == [np.ndarray, np.ndarray]):
+            raise TypeError("Dataset type should be of shape (np.ndarray, np.ndarray).")
+            
         self.dataset = dataset
-        # count of shadow training sets, must be eval
+
+        if not(isinstance(target_model, TensorFlowV2Classifier) or isinstance(target_model,PyTorchClassifier)):
```

Comment:
a space missed before `PyTorchClassifier. but I think no big deal after we do black reformat in the end